CHANGELOG.md

```diff
@@ -19,16 +19,19 @@ Release channels have their own copy of this changelog:
 
 ## [2.0.0]
 * Breaking
-  * SDK: Support for Borsh v0.9 removed, please use v1 or v0.10 (#1440)
-  * SDK: `Copy` is no longer derived on `Rent` and `EpochSchedule`, please switch to using `clone()` (solana-labs#32767)
-  * SDK: deprecated SyncClient trait methods removed
+  * SDK:
+    * Support for Borsh v0.9 removed, please use v1 or v0.10 (#1440)
+    * `Copy` is no longer derived on `Rent` and `EpochSchedule`, please switch to using `clone()` (solana-labs#32767)
+    * Deprecated symbols removed
+  * `solana-program`: deprecated symbols removed
```

Comment:
```suggestion
    * `solana-sdk`: Deprecated symbols removed
    * `solana-program`: deprecated symbols removed
```

i wonder if devs actually hold these separate (or whether the prefix in the change log is purely indicating the crate?)

Reply:
Good question. I'll take your suggestion nevertheless.
```suggestion
    * `solana-sdk`: deprecated symbols removed
    * `solana-program`: deprecated symbols removed
```